services/dask-sidecar/src/simcore_service_dask_sidecar/computational_sidecar/core.py

```diff
@@ -194,9 +195,15 @@ async def run(self, command: list[str]) -> TaskOutputData:
                 self._publish_sidecar_log,
             )
 
-            integration_version = await get_integration_version(
+            integration_version: version.Version = LEGACY_INTEGRATION_VERSION
+            progress_regexp: re.Pattern[str] = PROGRESS_REGEXP
+            if image_labels := await get_image_labels(
                 docker_client, self.docker_auth, self.service_key, self.service_version
-            )
+            ):
+                if iv := image_labels.integration_version:
+                    integration_version = version.Version(iv)
+                if pr := image_labels.progress_regexp:
```

Comment:
Q: can we have longer names here, please? I'm asking kindly to persuade you in changing them. :D

Reply:
Done